src/main/java/wolox/training/models/Book.java

```diff
@@ -25,6 +29,10 @@
  * @author luismiguelrodriguez
  */
 @Entity
+@Getter
```

Comment:
Why not setters? use `@Data`

Reply:
The setter methods are defined below in the entity, they use the pre conditions, please confirm me, if I should add them anyway

Reply:
corrected